src/Chessboard/helpers.js

```diff
@@ -136,12 +136,13 @@ function fenToPieceCode(piece) {
   if (piece === '-') {
     return '-';
```

Comment:
we don't have a piece or an icon for the hyphen. I think we need to somehow filter hyphens before calling this function

Reply:
ok, I removed it, increase it by mistake